_scandir.c

```diff
@@ -48,6 +48,11 @@ comment):
     PyUnicode_AsUnicode(unicode); *(addr_length) = PyUnicode_GetSize(unicode)
 #endif
 
+// Because on PyPy not worjing without
```

Comment:
Could you please fix the typo: worjing -> working

Reply:
Thanks. Fix on b476572